### PR TITLE
frequently-asked-questions.md converted into YML

### PR DESCRIPTION
**Pull request by v-hearya**
Per the [this page](https://review.docs.microsoft.com/en-us/help/contribute/contribute-how-to-faq-guide?branch=master), Updated following Markdown to YAML template, to improve the discoverability of the FAQ pages in Google.

### CODE REVIEW

File: docs/framework/data/adonet/sql/linq/frequently-asked-questions.yml [hunk 1]
```diff
@@ -0,0 +1,253 @@
+### YamlMime:FAQ
+metadata:
+  description: "Learn more about: Frequently Asked Questions"
+  title: "Frequently Asked Questions"
+  ms.date: "03/30/2017"
+  dev_langs:
+    - "csharp"
+    - "vb"
+  ms.assetid: 252ed666-0679-4eea-b71b-2f14117ef443
```
Comment (Youssef1313) [inline docs/framework/data/adonet/sql/linq/frequently-asked-questions.yml:9]: I think ms.assetid isn't used.

```suggestion
```

**Comment by IEvangelist** [comment]
Attention: @Youssef1313 -- seeing _build verifier_ failure here, not sure why?

File: docs/framework/data/adonet/sql/linq/frequently-asked-questions.yml [hunk 2]
```diff
@@ -0,0 +1,253 @@
...
+          I made a change to data in the database, but when I reran my application, the change was no longer there.
+        answer: |
+          Make sure that you call <xref:System.Data.Linq.DataContext.SubmitChanges%2A> to save results to the database.
+          
+  - name: "Database Connection: Open How Long?"
+    questions:
+      - question: |
+          How long does my database connection remain open?
+        answer: |
+          A connection typically remains open until you consume the query results. If you expect to take time to process all the results and are not opposed to caching the results, apply <xref:System.Linq.Enumerable.ToList%2A> to the query. In common scenarios where each object is processed only one time, the streaming model is superior in both `DataReader` and [!INCLUDE[vbtecdlinq](../../../../../../includes/vbtecdlinq-md.md)].
```
Comment (IEvangelist) [inline docs/framework/data/adonet/sql/linq/frequently-asked-questions.yml:38]: This `[!INCLUDE]` is not rendering correctly, I do not believe the that YML supports the include syntax like MD files do.

**Comment by Youssef1313** [comment]
@IEvangelist You can safely ignore this failure for now (the build won't fail on subsequent PRs). This is a case where a deleted file doesn't need a redirection that I'm not handling. I opened https://github.com/dotnet/docs-actions/issues/12 to track fixing this.

File: docs/framework/data/adonet/sql/linq/frequently-asked-questions.yml [hunk 3]
```diff
@@ -0,0 +1,253 @@
...
+    questions:
+      - question: |
+          Is a public provider model available?
+        answer: |
+          No public provider model is available. At this time, [!INCLUDE[vbtecdlinq](../../../../../../includes/vbtecdlinq-md.md)] supports SQL Server and SQL Server Compact 3.5 only.
+          
+  - name: SQL-Injection Attacks
+    questions:
+      - question: |
+          How is [!INCLUDE[vbtecdlinq](../../../../../../includes/vbtecdlinq-md.md)] protected from SQL-injection attacks?
```
Comment (IEvangelist) [inline docs/framework/data/adonet/sql/linq/frequently-asked-questions.yml:183]:
```suggestion
          How is LINQ to SQL protected from SQL-injection attacks?
```

We should replace ALL occurrences of `[!INCLUDE[vbtecdlinq](../../../../../../includes/vbtecdlinq-md.md)]` with `LINQ to SQL`.

**Review by IEvangelist — CHANGES_REQUESTED** [review]
Some issues with the includes...

image


image

File: docs/framework/data/adonet/sql/linq/frequently-asked-questions.yml (hunk 2 above)
Comment (v-hearya) [reply docs/framework/data/adonet/sql/linq/frequently-asked-questions.yml:37]: Under Question [!INCLUDE] doesn't support. We can use it in Answer or summary.

**Comment by v-hearya** [comment]
@IEvangelist - Neede changes made, please verify.

**Review by IEvangelist — APPROVED** [review]

**Pull request merged**